fieldservice_route/models/fsm_route_dayroute.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2019 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class FSMRouteDayRoute(models.Model):
+    _name = 'fsm.route.dayroute'
+
+    @api.multi
+    @api.depends('route_id')
+    def _compute_order_count(self):
+        for rec in self:
+            rec.order_count = 0
+            if rec.route_id and rec.route_id.order_ids:
+                rec.order_count = len(rec.route_id.order_ids)
+
+    person_id = fields.Many2one('fsm.person', string='Person')
+    route_id = fields.Many2one('fsm.route', string='Route')
+    date = fields.Date(string='Date')
+    team_id = fields.Many2one('fsm.team', string='Team')
+    stage_id = fields.Many2one('fsm.stage', string='Stage',
+                               domain="[('stage_type', '=', 'route')]")
+    territory_id = fields.Many2one('fsm.territory', string='Territory')
+    longitude = fields.Float("Longitude")
+    latitude = fields.Float("Latitude")
+    last_location_id = fields.Many2one('fsm.location', string='Last Location')
+    date_start_planned = fields.Datetime(string='Planned Start Time')
+    start_location_id = fields.Many2one('fsm.location', string='Start Location')
+    end_location_id = fields.Many2one('fsm.location', string='End Location')
+    work_time = fields.Float(string='Time before overtime (in hours)',
+                             default=8.0)
+    max_allow_time = fields.Float(string="Maximal Allowable Time (in hours)",
+                                  default=10.0)
+    order_count = fields.Integer(string='Number of Orders',
+                                 compute=_compute_order_count)
```

Comment:
@nikul-serpentcs Please move the code from here:
https://github.com/OCA/field-service/blob/12.0/fieldservice/models/fsm_route.py
to this module.

You forgot the name and `order_ids` fields, the method and the constraint.

Reply:
Done

Reply:
@murtuzasaleh Still missing the method:
https://github.com/OCA/field-service/blob/12.0/fieldservice/models/fsm_route.py#L31

and the constraint:
https://github.com/OCA/field-service/blob/12.0/fieldservice/models/fsm_route.py#L25

Reply:
Done